demo/demo.md

```diff
@@ -766,3 +766,54 @@ For use cases where the field is `required`, but live validation is not wanted,
   </auro-input>
   ```
 </auro-accordion>
+
+## Swapping Values Between Inputs
+
+<div class="exampleWrapper exampleWrapper--flex">
+<!-- AURO-GENERATED-CONTENT:START (FILE:src=./../apiExamples/swapValue.html) -->
+<!-- The below content is automatically added from ./../apiExamples/swapValue.html -->
+<auro-input id="swapExampleLeft" bordered>
+  <span slot="label">Left Input</span>
+</auro-input>
+<auro-button id="swapExampleBtn">Swap Values</auro-button>
+<auro-input id="swapExampleRight" bordered>
+  <span slot="label">Right Input</span>
+</auro-input>
+<!-- AURO-GENERATED-CONTENT:END -->
+</div>
+
+<auro-accordion lowProfile justifyRight>
+  <span slot="trigger">See code</span>
+
+<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../apiExamples/swapValue.html) -->
+<!-- The below code snippet is automatically added from ./../apiExamples/swapValue.html -->
+```html
+<auro-input id="swapExampleLeft" bordered>
+  <span slot="label">Left Input</span>
+</auro-input>
+<auro-button id="swapExampleBtn">Swap Values</auro-button>
+<auro-input id="swapExampleRight" bordered>
+  <span slot="label">Right Input</span>
+</auro-input>
+```
+<!-- The below code snippet is automatically added from ./../apiExamples/swapValue.html -->
+<!-- AURO-GENERATED-CONTENT:END -->
+<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../apiExamples/swapValue.js) -->
+<!-- The below code snippet is automatically added from ./../apiExamples/swapValue.js -->
+```js
+export function swapInputValues(selectors) {
+  const btn = document.querySelector(selectors[0]);
+  const inputOne = document.querySelector(selectors[1]);
+  const inputTwo = document.querySelector(selectors[2]);
+
+  btn.addEventListener('click', () => {
+    const valueOne = inputOne.value;
+    const valueTwo = inputTwo.value;
+
+    inputOne.value = valueTwo;
+    inputTwo.value = valueOne;
+  });
+}
+```
```

Comment:
There should be a comment in here that states that this optional javascript can be placed anywhere in their app.

Reply:
Resolving without direct change due to review comments being added to a draft PR that was not ready for review.